wasm-rpc/src/text.rs

```diff
@@ -5,7 +5,7 @@ use std::fmt::{Debug, Formatter};
 use wasm_wave::wasm::{WasmType, WasmTypeKind, WasmValue, WasmValueError};
 
 #[derive(Clone, PartialEq, Eq)]
```

Comment:
I think you should basically delete most of the code I wrote in this `text` module, and make it only contain `wasm_wave::wasm::WasmType` and `wasm_wave::wasm::WasmValue` instances for the new `TypeAnnotatedValue` type.

Reply:
Done